index.html

```diff
@@ -603,6 +603,18 @@ <h1>Terminology</h1>
         capabilities. It describes the <a>Properties</a>, <a>Actions</a>, and <a>Events</a> and common metadata that are 
         shared for an entire group of <a>Things</a>. Compared to a Thing Description, a Thing Model does not contain enough 
         information to identify or interact with a Thing instance.</dd>
+      <dt>
+      <dt>
+        <dfn>Partial TD</dfn>
+      </dt>
+      <dd>
+        A <a>Partial TD</a> is an object that follows the same hierarchical structure of the <a>TD</a> information model, 
+        but it may not contain all the mandatory elements. 
```

Comment:
The proposal looks good to me.

An alternative to
"but it may not contain all the mandatory elements. "
may be
"but it does not have to contain all mandatory elements."

I leave it to native speakers what is more clear....